src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  display: block;
+  margin-top: 20px;
+  font-weight: 300;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
Try to organize your CSS properties correctly
Please fix all cases
https://9elements.com/css-rule-order/